scripts/BLVO-EN/c101103000.lua

```diff
@@ -0,0 +1,88 @@
+--War Rock Mountain
+--Scripted by Kohana Sonogami
+function c101103000.initial_effect(c)
+	--Activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetCategory(CATEGORY_TOHAND+CATEGORY_SEARCH)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetCountLimit(1,101103000+EFFECT_COUNT_CODE_OATH)
+	e1:SetOperation(c101103000.activate)
+	c:RegisterEffect(e1)
+	--special summon
+	local e2=Effect.CreateEffect(c)
+	e2:SetDescription(aux.Stringid(101103000,1))
+	e2:SetCategory(CATEGORY_SPECIAL_SUMMON)
+	e2:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_TRIGGER_O)
+	e2:SetCode(EVENT_PHASE+PHASE_BATTLE_START)
+	e2:SetRange(LOCATION_FZONE)
+	e2:SetCountLimit(1,101103000)
+	e2:SetCondition(c101103000.spcon)
+	e2:SetTarget(c101103000.sptg)
+	e2:SetOperation(c101103000.spop)
+	c:RegisterEffect(e2)
+	--destroy replace
+	local e3=Effect.CreateEffect(c)
+	e3:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_CONTINUOUS)
+	e3:SetCode(EFFECT_DESTROY_REPLACE)
+	e3:SetRange(LOCATION_FZONE)
+	e3:SetTarget(c101103000.reptg)
+	e3:SetValue(c101103000.repval)
+	e3:SetOperation(c101103000.repop)
+	c:RegisterEffect(e3)
+end
+function c101103000.thfilter(c)
+	return c:IsSetCard(0x263) and c:IsType(TYPE_MONSTER) and c:IsAbleToHand()
+end
+function c101103000.activate(e,tp,eg,ep,ev,re,r,rp)
+	if not e:GetHandler():IsRelateToEffect(e) then return end
+	local g=Duel.GetMatchingGroup(c101103000.thfilter,tp,LOCATION_DECK,0,nil)
+	if #g>0 and Duel.SelectYesNo(tp,aux.Stringid(101103000,0)) then
+		Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_ATOHAND)
+		local sg=g:Select(tp,1,1,nil)
+		Duel.SendtoHand(sg,nil,REASON_EFFECT)
+		Duel.ConfirmCards(1-tp,sg)
+	end
+end
+function c101103000.cfilter1(c)
+	return c:IsFacedown() or not c:IsRace(RACE_WARRIOR)
+end
+function c101103000.spcon(e,tp,eg,ep,ev,re,r,rp)
+	return not Duel.IsExistingMatchingCard(c101103000.cfilte1r,tp,LOCATION_MZONE,0,1,nil)
+end
+function c101103000.cfilter2(c,code)
+	return c:IsFaceup() and c:IsCode(code)
+end
+function c101103000.spfilter(c,e,tp)
+	return c:IsSetCard(0x263) and c:IsType(TYPE_MONSTER) and c:IsCanBeSpecialSummoned(e,0,tp,false,false)
+		and not Duel.IsExistingMatchingCard(c101103000.cfilter2,tp,LOCATION_ONFIELD,0,1,nil,c:GetCode())
+end
+function c101103000.sptg(e,tp,eg,ep,ev,re,r,rp,chk)
+	if chk==0 then return Duel.GetLocationCount(tp,LOCATION_MZONE)>0
+		and Duel.IsExistingMatchingCard(c101103000.spfilter,tp,LOCATION_DECK,0,1,nil,e,tp) end
```

Comment:
This card should special summon from Hand not deck, please fix

Reply:
Also should be able to summon if you only control warriors not just if you have no monsters